docker/run_and_collect_cov.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+apt update && apt install -y lcov ggcov
```

Comment:
install `lcov` and `ggcov` should not be done here. This way they are done every CI. We want to pre-install them once.

Reply:
I thought so too... maybe modify the dockerfile for xenial build?

Reply:
yes. need to modify, re-run docker, and push new version to docker hub. What are the extra packages you use?

Reply:
just lcov and gcov I believe, CI uses gcc as compiler so llvm-cov is not required